src/demo/DemoTestLoanService.java

```diff
@@ -2269,43 +2270,29 @@ private static void testBulkReverseLoanTransactions() throws MambuApiException {
 		return loanTransactions;
 	}
 
+	// TODO once with V4.4 this method needs to be deleted
 	/**
-	 * Tests PATCHing the setExpectedDisbursementDate and setFirstRepaymentDate on a loan account.
+	 * Updates expected disbursement date and first repayment date
 	 * 
+	 * @param loanAccount
+	 *            The loan account to be updated
 	 * @throws MambuApiException
 	 */
-	private static void testPatchLoanAccountDisbursementAndFirstRepaymentDate() throws MambuApiException {
-
-		methodName = new Object() {}.getClass().getEnclosingMethod().getName();
-		System.out.println("\nIn : " + methodName);
-
-		LoansService loanService = MambuAPIFactory.getLoanService();
-
-		LoanAccount theAccount = newAccount;
-		LoanAccount account = new LoanAccount();
-
-		account.setId(theAccount.getId());
+	private static LoanAccount updateExpectedDisbursementDateAndFirstRepaymentDate(LoanAccount loanAccount)
+			throws MambuApiException {
 
+		LoanAccount updatedLoanAccount = loanAccount;
 		Calendar currentCalendar = Calendar.getInstance();
 		currentCalendar
-				.setTime(theAccount.getDisbursementDate() != null ? theAccount.getDisbursementDate() : new Date());
+				.setTime(loanAccount.getDisbursementDate() != null ? loanAccount.getDisbursementDate() : new Date());
 		currentCalendar.add(Calendar.DAY_OF_MONTH, 1);
```

Comment:
Let's suppose that the disbursement date is not null on the loan account. In this case the currentCalendar value will be equal to the corresponding value from the loanAccount (which is fine), but after that, we add a day to the currentCalendar value. What will happen in this case when trying to PATCH the loan account in Mambu? We'll get an INCONSISTENT_DISBUSEMENT_DETAILS_WITH_ACCOUNT error, as described in MBU-13945?

The same question for adding two days to the first repayment date, because we set it twice, also - in testPatchLoanAccountTerms() on the disbursementDetails level and in here, on the loan account level.